tests/Tests/600__ai_apps/650__rhosak/650__rhosak.robot

```diff
@@ -1,23 +1,60 @@
 *** Settings ***
 Resource        ../../../Resources/Page/LoginPage.robot
 Resource        ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Resource        ../../../Resources/Page/HybridCloudConsole/HCCLogin.robot
 Library         SeleniumLibrary
+Library         OpenShiftCLI
 Suite Setup     Kafka Suite Setup
 Suite Teardown  Kafka Suite Teardown
+Test Setup      Kafka Test Setup
+
+*** Variables ***
+${rhosak_real_appname}=  rhosak
+${rhosak_displayed_appname}=  OpenShift Streams for Apache Kafka
 
 *** Test Cases ***
 Verify RHOSAK Is Available In RHODS Dashboard Explore Page
   [Tags]  ODS-258  Smoke  Sanity
-  Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
-  Login To RHODS Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
-  Wait for RHODS Dashboard to Load
-  Verify Service Is Available In The Explore Page    OpenShift Streams for Apache Kafka
-  Verify Service Provides "Get Started" Button In The Explore Page    OpenShift Streams for Apache Kafka
-  Verify Service Provides "Enable" Button In The Explore Page    OpenShift Streams for Apache Kafka
+  Verify Service Is Available In The Explore Page    ${rhosak_displayed_appname}
+  Verify Service Provides "Get Started" Button In The Explore Page    ${rhosak_displayed_appname}
+  Verify Service Provides "Enable" Button In The Explore Page    ${rhosak_displayed_appname}
+
+Verify User Can Enable RHOSAK from Dashboard Explore Page
+  [Tags]  Sanity  Smoke
+  ...     ODS-392
+  Enable RHOSAK
+  Wait Until Page Contains Element   xpath://div[@class='pf-c-alert pf-m-success']
+  Capture Page Screenshot  kafka_enable_msg.png
+  Menu.Navigate To Page    Applications    Enabled
+  Wait Until RHODS Dashboard JupyterHub Is Visible
+  Page Should Contain Element  xpath://div[@class="pf-c-card__title"]/span[.="${rhosak_displayed_appname}"]
+  Capture Page Screenshot  kafka_enable_tab.png
+  Launch OpenShift Streams for Apache Kafka From RHODS Dashboard Link
+  Login to HCC  ${SSO.USERNAME}  ${SSO.PASSWORD}
+  Wait For HCC Splash Page
+  Maybe Skip RHOSAK Tour
+  Wait Until Page Contains    Kafka Instances
+  Delete Configmap    name=rhosak-validation-result  namespace=redhat-ods-applications
 
-* Keywords ***
+** Keywords ***
 Kafka Suite Setup
   Set Library Search Order  SeleniumLibrary
 
 Kafka Suite Teardown
   Close All Browsers
+
+Kafka Test Setup
+  Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+  Login To RHODS Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+  Wait for RHODS Dashboard to Load
+
+Enable RHOSAK
```

Comment:
Wouldn't it be better to have this keyword in HCCLogin or other resource file?

Reply:
yes. In a future refactor, I would gather in a Resource subfolder all the Keywords needed in AI Apps components. Is that okay to postpone this change?

Reply:
sounds good to me